docs/zh-hant/docs/deployment/cloud.md

```diff
@@ -0,0 +1,17 @@
+# 在雲端部署 FastAPI
+
+你幾乎可以使用 **任何雲端供應商** 來部署你的 FastAPI 應用程式。
+
+在大多數情況下，主要的雲端供應商都有部署 FastAPI 的指南。
+
+## 雲端供應商 - 贊助商
+
+一些雲端供應商 ✨ [**贊助 FastAPI**](../help-fastapi.md#sponsor-the-author){.internal-link target=_blank} ✨，這確保了 FastAPI 及其 **生態系統** 持續健康地 **發展**。
+
+這也展現了他們對 FastAPI 和其 **社群**（包括你）的真正承諾，他們不僅希望為你提供 **優質的服務**，還希望確保你擁有一個良好且健康的框架：FastAPI。🙇
```

Comment:
```suggestion
這也展現了他們對 FastAPI 和其 **社群**（包括你）的真正承諾，他們不僅希望為你提供 **優質的服務**，還希望確保你擁有一個**良好且健康的框架**：FastAPI。🙇
```

Reply:
Thanks for the suggestion! I added a space before the bold text to ensure that the markdown is properly rendered.